courts_db/data/courts.json

```diff
@@ -3323,7 +3383,8 @@
         "name_abbreviation": null,
         "regex": [
             "Florida ${sup}",
-            "${sup} Florida"
+            "${sup} Florida",
+            "Court of Florida, Division B"
```

Comment:
this is likely not the Supreme Court.  Im not sure what division B is but we probably need to identify what it is